### PR TITLE
chore(docs): bundler setup more readable - installation page

### PR DESCRIPTION
**Pull request by kavinvalli**

### CODE REVIEW

**Comment by Mokshit06** [comment]
Cab we sync the tabs of the two code blocks (one with npm command and the file)? Currently if i have to switch to esbuild I'll have to change both the tabs.

**Comment by kavinvalli** [comment]
> Can we sync the tabs of the two code blocks (one with npm command and the file)? Currently if i have to switch to esbuild I'll have to change both the tabs.

Hm, let me look into it

**Comment by kavinvalli** [comment]
@Mokshit06 Converted to tabs

File: site/src/pages/docs/docs-layout.tsx [hunk 1]
```diff
@@ -3,6 +3,16 @@ import { globalStyle, style } from '@macaron-css/core';
...
 import { Link } from '../../../renderer/Link';
 
+globalStyle('blockquote', {
+  padding: '0.8rem',
+  borderLeft: '2px solid #ff4089',
+  background: '#ffffff10',
+});
+
+globalStyle('blockquote p', {
+  fontSize: '0.8rem !important',
```
Comment (Mokshit06) [inline site/src/pages/docs/docs-layout.tsx:13]: Increase the font size to be the same as rest

File: site/src/pages/docs/installation.page.mdx [hunk 2]
```diff
@@ -30,13 +30,22 @@ yarn add @macaron-css/core @macaron-css/react
 
 Macaron currently supports vite and esbuild. Install and setup the plugin for your bundler :-
 
-### Vite
+<CH.Spotlight className="bundler" start={1}>
 
-```fish
+```
+Macaron currently supports vite and esbuild.
+```
```
Comment (Mokshit06) [inline site/src/pages/docs/installation.page.mdx:37]: Replace this with
\`\`\`fish
\`\`\`
This way there won't be any empty space before the "Installation" tab

File: site/src/pages/docs/docs-layout.tsx (hunk 1 above)
Comment (kavinvalli) [reply site/src/pages/docs/docs-layout.tsx:13]: It doesn't look that good that way
This is normal font size:
Screenshot 2022-12-06 at 14 12 51
This is `0.8rem`:
Screenshot 2022-12-06 at 14 13 19

Comment (Mokshit06) [reply site/src/pages/docs/docs-layout.tsx:13]: hm, `0.8rem` looks too small. maybe `1rem`?

Comment (kavinvalli) [reply site/src/pages/docs/docs-layout.tsx:13]: Screenshot 2022-12-06 at 14 39 00

Comment (Mokshit06) [reply site/src/pages/docs/docs-layout.tsx:13]: yeah this looks good

**Comment by Mokshit06** [comment]
lgtm thanks!

**Pull request merged**